packages/main/src/RPA/JSON.py

```diff
@@ -24,21 +109,43 @@ def __init__(self):
 
     @keyword("Load JSON from file")
     def load_json_from_file(self, filename: str) -> Any:
-        """Load JSON data from a file.
+        """Load JSON data from a file, and return it as JSON object.
+        Depending on the input file the object can be either a dictionary,
+        a list, or a scalar value.
+
+        :param filename: path to input file
+
+        Example:
+
+        .. code:: robotframework
+
+           &{auth}=    Load JSON from file    auth.json
+           Log   Current auth token: ${auth.token}
 
-        :param filename: path to file
-        :return: json as a dictionary object
         """
         self.logger.info("Loading JSON from file: %s", filename)
         with open(filename) as json_file:
             return json.load(json_file)
 
     @keyword("Save JSON to file")
     def save_json_to_file(self, doc: Any, filename: str) -> None:
```

Comment:
I would hint the type like 
```
doc: Union[str, dict]
```
but not sure if in the scope of this PR.

Reply:
You can of course define type aliases, but you can also enjoy this long conversation: https://github.com/python/typing/issues/182

It gets... less useful when you support all JSON things like plain scalar values as root objects.

Reply:
Maybe we can go with: `JSON = Union[str, int, float, bool, None, Dict[str, Any], List[Any]]`

Reply:
Yeah, I was afraid it would lead to something like the linked discussion.